### ISSUE TITLE
[redesign-2021] - Homepage

### ISSUE THREAD

**Issue by chrisguindon** [EclipseFdn/ecdtools.eclipse.org#104]
As previously noted, the new design is available here:
#91

Homepage:
https://projects.invisionapp.com/share/5C10DOVC3QTG#/screens

**Deliverables:**

- [x] Jumbotron
- [x] Powered by participation section
- [x] Webinar Series Section
- [x] Featured Projects
- [x] What's new?
- [x] Open Source Community-driven Collaboration // Highlights
- [x] Members

**Comment by ericpoirier** [comment]
Hi,

Currently working on the redesign of the home page and i'm now looking at the What's new section. Just to make it clear, i'm going to display the most recent items news, blog, video and event items in that section. The 4 sections and their icons will be hardcoded but their content will be more dynamic.

Also for the Mailing List section, we are currently not using a form to add new emails to mailing lists. We are currently adding a link to the following page[1] for users to add the mailing list to their Eclipse Account.
[1] - https://accounts.eclipse.org/mailing-list/ecd-tools-wg

Let me know if these are raising any concerns.

Thanks!

**Comment by croundy** [comment]
@ericpoirier You've got the right idea on the "What's New" section. No concerns about handling the mailing list as you describe. Thanks! 


> Hi,
> 
> Currently working on the redesign of the home page and i'm now looking at the What's new section. Just to make it clear, i'm going to display the most recent items news, blog, video and event items in that section. The 4 sections and their icons will be hardcoded but their content will be more dynamic.
> 
> Also for the Mailing List section, we are currently not using a form to add new emails to mailing lists. We are currently adding a link to the following page[1] for users to add the mailing list to their Eclipse Account.
> [1] - https://accounts.eclipse.org/mailing-list/ecd-tools-wg
> 
> Let me know if these are raising any concerns.
> 
> Thanks!

**Comment by ericpoirier** [comment]
Hi guys (@croundy @brianking),

Here is a preview of where i'm at with the home page.

https://deploy-preview-124--eclipsefdn-ecdtools.netlify.app/

I think I'm pretty close to be done with the home page. Keep in mind that I didn't really look at the mobile view yet but if you have any feedback, please feel free to let me know.

Let me know what you guys think, if you have any concerns or changes you'd like to make.

Thanks!

**Comment by croundy** [comment]
@ericpoirier coming along nicely. A few minor things that I noticed:

1. Jumbotron. If possible, would like "Open Source" to be part of the 2nd line of text
2. Rotating banner: Minor nit.... the word "talk" is on its own line.
3. Thinking that the View all Projects button should be directed at the Projects page, rather than show all projects on the home page 
4. The "What's New" section references IoT assets rather than ECD Tools. Not a big deal ATM, but just to be sure we point to the right articles, videos, blogs, and calendars.
5. The banner section that says "Open Source Community-driven Collaboration" is supposed to have a parallax effect.    
6. Question about the Cloud ToolTime banners having a date on them. Wondering if it makes sense to make at least one of them (probably the one on the bottom) more generic so we don't have to change it out with every new episode and make the rotating banner specific to the next episode. This does mean we will need to change that every 3 weeks or so. 

Anyway, excellent first pass....thanks so much!

**Comment by brianking** [comment]
I tried in responsive design mode, and there are some issues. The major ones are:

- main sections images are not responsive, you just see the left section of an image
- bad wrapping / alignment in Webinar Series section
- not enough padding around some buttons when staked over each other, e.g. 'Register Now' and 'Join the Conversation'
- poor alignment of head-shot image and title/name in Projects boxes (tablet mode)

Looks great in general!

**Comment by ericpoirier** [comment]
> @ericpoirier coming along nicely. A few minor things that I noticed:
> 
> 1. Jumbotron. If possible, would like "Open Source" to be part of the 2nd line of text
+1

> 2. Rotating banner: Minor nit.... the word "talk" is on its own line.
+1
> 3. Thinking that the View all Projects button should be directed at the Projects page, rather than show all projects on the home page
+1
> 4. The "What's New" section references IoT assets rather than ECD Tools. Not a big deal ATM, but just to be sure we point to the right articles, videos, blogs, and calendars.

Note that these items are items that I copied from another website until we have actual news items in here. 

> 5. The banner section that says "Open Source Community-driven Collaboration" is supposed to have a parallax effect.
+1
> 6. Question about the Cloud ToolTime banners having a date on them. Wondering if it makes sense to make at least one of them (probably the one on the bottom) more generic so we don't have to change it out with every new episode and make the rotating banner specific to the next episode. This does mean we will need to change that every 3 weeks or so.

Not too sure what you mean here.

> 
> Anyway, excellent first pass....thanks so much!

**Comment by ericpoirier** [comment]
I did most of the changes from @croundy and @brianking.

You can see those changes here:
https://deploy-preview-130--eclipsefdn-ecdtools.netlify.app/

**Comment by brianking** [comment]
@ericpoirier All the responsive issues are resolved, except for the first of an image not being centered correctly. It is the one with the man with the headphones.

<img width="414" alt="Screenshot 2021-05-24 at 16 33 37" src="https://user-images.githubusercontent.com/114116/119371022-e3137780-bcad-11eb-8f60-b807a9ee57ad.png">

**Comment by ericpoirier** [comment]
Hi @brianking I updated the image position for mobile.

https://deploy-preview-130--eclipsefdn-ecdtools.netlify.app/

Let me know if there's anything else.

Thanks!

**Comment by croundy** [comment]
![image](https://user-images.githubusercontent.com/72274403/119683440-e2d7c100-be00-11eb-8eaf-01a061ffe1d1.png)> > 2. Question about the Cloud ToolTime banners having a date on them. Wondering if it makes sense to make at least one of them (probably the one on the bottom) more generic so we don't have to change it out with every new episode and make the rotating banner specific to the next episode. This does mean we will need to change that every 3 weeks or so.
> 
> Not too sure what you mean here.

Sorry if my message was a bit cryptic. I'm referring to this as the rotating banner, as we will use it to promote multiple things. I would expect to have 3-5 different banners at any given time. So, whatever Cloud Tool Time banner we implement here should be promoting the next upcoming webinar, with image of the presenter, name of the talk, date, etc.:
 
![image](https://user-images.githubusercontent.com/72274403/119682680-50372200-be00-11eb-94fe-4676e55752b6.png)

The footer on the bottom is more static, so we should make it more generic, eliminate any dates or specific topics being discussed. In this case it says December 2020:

![image](https://user-images.githubusercontent.com/72274403/119683440-e2d7c100-be00-11eb-8eaf-01a061ffe1d1.png)

**Comment by ericpoirier** [comment]
> > > 1. Question about the Cloud ToolTime banners having a date on them. Wondering if it makes sense to make at least one of them (probably the one on the bottom) more generic so we don't have to change it out with every new episode and make the rotating banner specific to the next episode. This does mean we will need to change that every 3 weeks or so.
> > 
> > 
> > Not too sure what you mean here.
> 
> Sorry if my message was a bit cryptic. I'm referring to this as the rotating banner, as we will use it to promote multiple things. I would expect to have 3-5 different banners at any given time. So, whatever Cloud Tool Time banner we implement here should be promoting the next upcoming webinar, with image of the presenter, name of the talk, date, etc.:

I think it would probably be a good idea to involve Marketing here to design a banner containing all those elements. Unless you want to use the current layout but just change the text and image. Also I was thinking of using data files for every banner in the carousel, this way we don't need to write custom html every time we add or remove banners which would be time consuming.

**Comment by ericpoirier** [comment]
The redesign of the home page is now live on production.

All the redesign elements have been implemented.

One thing to note is the implementation for the carousel section. At first in the design it was only displaying a webinar but it's now showing featured story items which have been hardcoded in ecdtools for now.

For the moment we are not supporting images for the featured story API but I created a bug[1] today to take care of that. So the end goal here would be to use the featured story API to populate the carousel section rather then hardcoded content.

I'm going to open a new issue to track the update of the carousel.

[1] - https://foundation.eclipse.org/infrazilla/show_bug.cgi?id=4349
